src/main/java/io/kestra/plugin/aws/AbstractConnectionInterface.java

```diff
@@ -76,4 +93,144 @@ public interface AbstractConnectionInterface {
     )
     @PluginProperty(dynamic = true)
     String getEndpointOverride();
+
+    @PluginProperty(dynamic = true)
+    default Boolean getCompatibilityMode() {
+        return false;
+    }
+
+    static String renderStringConfig(final RunContext runContext, final String config) throws IllegalVariableEvaluationException {
+        return Optional.ofNullable(config).map(Rethrow.throwFunction(runContext::render)).orElse(null);
```

Comment:
You can pass null to the render function, it'll return null so this seems useless

Reply:
it was already there but yes I agree, I will remove that method as it's a no gain over calling render directly